src/v/pandaproxy/schema_registry/service.cc

```diff
@@ -188,6 +191,45 @@ ss::future<> service::do_start() {
     });
 }
 
+ss::future<>
+create_acls(cluster::security_frontend& security_fe, ss::abort_source& as) {
+    // Initialize with any non-success errc
+    std::vector<cluster::errc> err_vec{cluster::errc::timeout};
+
+    auto create_acls_backoff = initial_create_acls_backoff;
+    std::vector<security::acl_binding> princpal_acl_binding{
+      security::acl_binding{
+        security::resource_pattern{
+          security::resource_type::topic,
+          model::schema_registry_internal_tp.topic,
+          security::pattern_type::literal},
+        security::acl_entry{
+          principal,
+          security::acl_host::wildcard_host(),
+          security::acl_operation::all,
+          security::acl_permission::allow}}};
+
+    while (err_vec[0] != cluster::errc::success) {
```

Comment:
issue: Should the abort source be checked here to break out of the loop? , e.g.: ` && !as.abort_requested()`